docs/guides/guides/screenshots-and-videos.mdx

```diff
@@ -58,14 +58,12 @@ If enabled, Cypress records a video for each spec file when running tests during
 }
 ```
 
+:::cypress-config-example
```

Comment:
this was missing from the video configuration example which broke the others which I missed in the previous PR